content/influxdb/v2.0/reference/cli/influx/v1/auth/list.md

```diff
@@ -0,0 +1,39 @@
+---
+title: influx v1 auth list
+description: >
+  The `influx v1 auth list` command lists and searches authorizations in the InfluxDB 1.x compatibility API.
+menu:
+  influxdb_2_0_ref:
+    name: influx v1 auth list
+    parent: influx v1 auth
+weight: 101
+influxdb/v2.0/tags: [authorization]
+---
+
+The `influx v1 auth list` command lists and searches authorizations in the [InfluxDB 1.x compatibility API](/influxdb/v2.0/reference/api/influxdb-1x/).
+
+## Usage
+```
+influx v1 auth list [flags]
+```
+
+#### Aliases
+`list`, `ls`, `find`
+
+## Flags
+| Flag |                   | Description                                                             | Input type | {{< cli/mapped >}}      |
+|:-----|:------------------|:------------------------------------------------------------------------|:----------:|:------------------------|
+| `-c` | `--active-config` | Config name to use for command                                          | string     | `$INFLUX_ACTIVE_CONFIG` |
```

Comment:
Does this list all authorizations for the specified active configuration?

Reply:
See [comment above](https://github.com/influxdata/docs-v2/pull/1768#discussion_r520281773), which I believe applies here.